object_tracker/labels.go

```diff
@@ -15,7 +14,7 @@ import (
 // GetTimestamp will retrieve and format a timestamp to be YYYYMMDD_HHMMSS
 func GetTimestamp() string {
 	currTime := time.Now()
-	return fmt.Sprintf(currTime.Format("20060102_150405"))
+	return string(currTime.Format("20060102_150405"))
```

Comment:
This is a cool way of specifying what format you want something in! Didn't know about it